src/main/java/com/google/sps/data/SampleData.java

```diff
@@ -172,4 +172,16 @@ public static void addToStudentScheduledSessionsByEmail(String email, TutorSessi
         students.add(student);
     }
 
+    public static void rateTutorByEmail(String tutorEmail, String studentEmail, int newRating) {
+        Tutor tutor = getTutorByEmail(tutorEmail);
+        Student student = getStudentByEmail(studentEmail);
+
+        tutors.remove(tutor);
+        tutor.rateTutor(newRating);
+        tutors.add(tutor);
```

Comment:
Do you have to remove the tutor and add again? The tutor object should get updated in the `tutors` list when you call `rateTutor` on the tutor object.